azure-pipelines.yml

```diff
@@ -169,23 +169,30 @@ stages:
         git submodule update --init
       displayName: 'Fetch submodules'
     - script: |
+            # For when updating this job to Meson: the `openblas` used here is
+            # missing pkg-config files. Ubunty Jammy (LTS 22.04) has openblas
+            # 3.20, so if that Docker image is available by then, just remove
+            # the unusual install method. For the bug report about missing .pc
+            # files: https://github.com/MacPython/openblas-libs/issues/74
             set -euo pipefail
             docker pull i386/ubuntu:bionic
             docker run -v $(pwd):/scipy i386/ubuntu:bionic /usr/bin/linux32 /bin/bash -c "cd scipy && \
             apt-get -y update && \
             apt-get -y install curl python3.8-dev python3.8 python3-distutils pkg-config libpng-dev libjpeg8-dev libfreetype6-dev && \
             curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
             python3.8 get-pip.py && \
+            pip3 install pip==22.0.4 && \
             pip3 --version && \
-            pip3 install setuptools==59.6.0 wheel numpy==1.18.5 cython==0.29.18 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath pythran==0.10.0 --user && \
+            pip3 install setuptools==59.6.0 wheel numpy==1.18.5 cython==0.29.21 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath pythran==0.10.0 && \
             apt-get -y install gcc-5 g++-5 gfortran-8 wget && \
             cd .. && \
             mkdir openblas && cd openblas && \
             target=\$(python3.8 ../scipy/tools/openblas_support.py) && \
             cp -r \$target/lib/* /usr/lib && \
             cp \$target/include/* /usr/include && \
             cd ../scipy && \
-            CC=gcc-5 CXX=g++-5 F77=gfortran-8 F90=gfortran-8 pip3 install . && \
+            CC=gcc-5 CXX=g++-5 F77=gfortran-8 F90=gfortran-8 pip3 install numpy && \
+            CC=gcc-5 CXX=g++-5 F77=gfortran-8 F90=gfortran-8 python3.8 setup.py install && \
```

Comment:
This is the change that's causing the `PyInit__direct` issue (moved, this is the 32-bit job definition).

Reply:
Fixed in `main` now, looked like an `importlib` bug.